lib/dns.js

```diff
@@ -278,7 +313,31 @@ exports.getServers = function() {
 };
 
 
-exports.setServers = function(servers) {
+exports.setServers = function setServers(servers) {
```

Comment:
Throwing asynchronously should be considered a bug (or at least very undesirable) IMO.